src/components/common/Loader/Loader.tsx

```diff
@@ -0,0 +1,19 @@
+import { useContext } from 'react';
+import { StateContext } from '../../../store';
+import './Loader.scss';
+
+export const Loader = () => {
+  const {
+    common: { isLoading },
+  } = useContext(StateContext);
+
+  return (
+    <>
+      {isLoading && (
```

Comment:
better to use this condition in the place where u render Loader, but not inside the Loader component